resources/docs/cache-in-ci.md

```diff
@@ -38,10 +38,12 @@ On GitHub Actions, you can use the [built-in cache action](https://github.com/ac
       - name: Rector Cache
         uses: actions/cache@v3
         with:
-          path: ./var/cache/rector
+          path: /tmp/rector
           key: ${{ runner.os }}-rector-${{ github.run_id }}
           restore-keys: ${{ runner.os }}-rector-
 
+      - run: mkdir -p /tmp/rector
```

Comment:
should it run before line 38 above executed?

Reply:
Nope. Needs to be here.